install.sh

```diff
@@ -96,14 +96,6 @@ INSTALL_DIR="/opt/ttn-gateway"
 if [ ! -d "$INSTALL_DIR" ]; then mkdir $INSTALL_DIR; fi
 pushd $INSTALL_DIR
 
-# Remove WiringPi built from source (older installer versions)
```

Comment:
This removal part is still needed because we need to keep the already installed gateways clean